src/tree/render.js

```diff
@@ -325,7 +176,7 @@ const renderTree = async ({ element, layout, app, model }) => {
     // Using the treemap created
     const allNodes = treemap(nodes);
     const activeNode = allNodes.data.id;
-    reRenderTree({ svg, data, allNodes, activeNode, o, width, height, treemap });
+    reRenderTree({ svg, divBox, data, allNodes, activeNode, o, width, height, treemap });
```

Comment:
Do we need to send all these? `allNodes` and `activeNode` might as well be created inside `reRenderTree` so we can pass fewer arguments?

Reply:
they are passed along when you re-render the tree, so they still need to be arguments. But I think that part can be optimized somehow, especially when I looked at selections. But for now, they can stay.